website/static/js/mailingListTreebeard.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+var $ = require('jquery');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+var $tb = require('js/treebeardHelpers');
+var projectSettingsTreebeardBase = require('js/projectSettingsTreebeardBase');
+
+function subscribe(item, notification_type, reload) {
+    var id = item.data.node.id;
+    var url = '/api/v1/project/' + id + '/mailing_list/';
+    var method = notification_type === 'enabled' ? 'POST' : 'DELETE';
```

Comment:
I'm having trouble finding where the `get_node_mailing_list` method is used- this seems to be the only function that references the URL, and this function only does POST and DELETE requests (not `GET`).  Can you clarify?